content/sensu-go/5.15/guides/use-federation.md

```diff
@@ -0,0 +1,335 @@
+---
+title: "Multi-cluster visibility with federation"
+linkTitle: "Reaching Multi-cluster Visibility"
+description: "In this guide, you'll learn how federate Sensu clusters and use a single web UI to access resources across those disparate clusters."
+weight: 400
+version: "5.15"
+product: "Sensu Go"
+platformContent: False
+menu: 
+  sensu-go-5.15:
+    parent: guides
+---
+
+- [What you can do with federation](#what-you-can-do-with-federation)
+- [Configure federation](#configure-federation)
+  - [Step 1 Configure backends for TLS](#step-1-configure-backends-for-tls)
+  - [Step 2 Configure shared token signing keys](#step-2-configure-shared-token-signing-keys)
+  - [Step 3 Add a cluster role binding and user](#step-3-add-a-cluster-role-binding-and-user)
+  - [Step 4 Create etcd replicators](#step-4-create-etcd-replicators)
+  - [Step 5 Register clusters](#step-5-register-clusters)
+    - [Register a single cluster](#register-a-single-cluster)
+    - [Register additional clusters](#register-additional-clusters)
```

Comment:
For some reason these two aren't being indented, or maybe that's the intention. I took a look and I don't see why they wouldn't be, but thought I'd point it out.

Reply:
Hmmmm maybe we can only use 2 levels even in these TOCs? I can't seem to find another doc that has 3 levels.

Reply:
> Hmmmm maybe we can only use 2 levels even in these TOCs? I can't seem to find another doc that has 3 levels.

I think this is a "known" limitation of Hugo, or our Hugo theme, which we have forgotten about 😅

Reply:
Yeah, I realized we have dealt with this in other docs with pipes. I'll make a suggestion to fix it.